### ISSUE TITLE
[wimpsim-reader] switch to astro (Trac #1342)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1334]
switch from coordinate-service to astro

this should not concern key components so much than test scripts

Migrated from https://code.icecube.wisc.edu/ticket/1342
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:13:35", 
    "description": "switch from coordinate-service to astro\n\nthis should not concern key components so much than test scripts", 
    "reporter": "mzoll", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067215093672", 
    "component": "combo simulation", 
    "summary": "[wimpsim-reader] switch to astro", 
    "priority": "normal", 
    "keywords": "coordinate-service astro", 
    "time": "2015-09-12T13:53:46", 
    "milestone": "", 
    "owner": "mzoll", 
    "type": "task"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150912T13:54:24: mzoll changed summary from "[wimpsim-reader]" to "[wimpsim-reader] switch to astro"

**Comment by icecube-trac** [comment]
Trac update at 20150922T01:09:54: olivas changed milestone from "IceSim 4.3" to "IceSim 5.1"

**Comment by icecube-trac** [comment]
Trac update at 20150925T08:14:06: 
* mzoll commented:

> In changeset 137953:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="137953"
> adapted rest of things to new interface; closes #1334
> ```


* mzoll changed status from "new" to "closed"
* mzoll changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:13:35: 
* olivas commented:

> Milestone IceSim 5.1 deleted


* olivas changed milestone from "IceSim 5.1" to ""
